lmdb.yaml

```diff
@@ -23,21 +22,25 @@ pipeline:
       tag: LMDB_${{package.version}}
       expected-commit: ce201088de95d26fc0da36ba805bf2ddc2ba74ff
 
+  - uses: patch
+    with:
+      patches: 0001-Debian-quirks-to-make-shared-library-versioned.patch
+
   - runs: |
       cd libraries/liblmdb
 
       cat > lmdb.pc << EOF
```

Comment:
fwiw, if you just use `cat >lmbd.pc << "EOF"` this would work.

i think i'd prefer that to more shell escaping.

Reply:
TIL!

Reply:
>  If any part of word is quoted, the delimiter is the result of quote removal on word, and the lines in the here-document are not expanded. If word is unquoted, all lines of the here-document are subjected to parameter expansion, command substitution, and arithmetic expansion, the character sequence \newline is ignored, and ‘\’ must be used to quote the characters ‘\’, ‘$’, and ‘`’.

from https://www.gnu.org/software/bash/manual/bash.html#Here-Documents

Reply:
shell!